cdk_framework/EKS/lib/app.ts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import { readFileSync} from 'fs';
+import { aws_eks as eks, aws_ec2 as ec2} from 'aws-cdk-lib';
+import { ClusterStack } from './stacks/cluster-stack';
+import { deployClusters } from './cluster-deployment';
+const yaml = require('js-yaml')
+
+
+
+
+
+
+
+const app = new cdk.App();
+
+const route = process.env.CDK_CONFIG_PATH ||  __dirname + '/config/clusters.yml';
+const raw = readFileSync(route)
+const configData = yaml.load(raw)
```

Comment:
I did that because I actually thought it made sense to decouple the concept of deploying clusters with a specific config file. I got this idea from testing, because in the testing, I wanted to call the deployClusters method, but I couldn't do it if the method was reading the config file. Instead, I have the file passed in as an argument, so you could decide what clusters to configure.